### DISCUSSION TITLE
Zigbee Curtain Module

### DISCUSSION THREAD

**Discussion by PacmanForever** [Koenkk/zigbee2mqtt#5631] [Devices]
Hi,

I can't find this device in the supported devices list: qs-zigbee-c01

But there are another devices starting with "qs-zigbee". 

Could someone tell me if it is supported?

Thanks

[
![imagen](https://user-images.githubusercontent.com/37216513/104023336-cf191e80-51c1-11eb-8a7c-020098524ebe.png)
](url)

**Comment by Koenkk** [comment]
I think it is, has been added recently and only available in the dev branch now. Will be supported in the 1 February release.

**Reply by PacmanForever** [reply]
Thanks!!
